Jenkinsfile

```diff
@@ -149,7 +149,7 @@ pipeline {
                 }
                 stage('Distribution tests') {
                     agent { label "distribution-tests" }
-                    steps { 
```

Comment:
Atom automatically removes these extra spaces when I save files. Sean, do you mind if we get rid of these extra spaces? With clang-format I can turn the 'removing extra spaces' feature off in Atom if this is an annoyance for you

Reply:
Definitely don't mind. Diffs are easy to see without whitespace changes by appending `?w=1` to the URL :)